### PR TITLE
Organizes context menus for database projects tree

### PR DESCRIPTION
**Pull request by Benjin**
Also stubs out some incoming functionality

### CODE REVIEW

File: extensions/sql-database-projects/package.json [hunk 1]
```diff
@@ -119,36 +139,78 @@
...
+          "command": "sqlDatabaseProjects.properties",
+          "when": "false"
         }
       ],
       "view/item/context": [
         {
-          "command": "sqlDatabaseProjects.close",
-          "when": "view == sqlDatabaseProjectsView"
+          "command": "sqlDatabaseProjects.build",
+          "when": "view == sqlDatabaseProjectsView",
```
Comment (Benjin) [inline extensions/sql-database-projects/package.json:162]: figure out `when` conditionals in this PR

File: extensions/sql-database-projects/package.json [hunk 2]
```diff
@@ -119,36 +139,78 @@
...
         },
         {
-          "command": "sqlDatabaseProjects.newScript",
-          "when": "view == sqlDatabaseProjectsView"
+          "command": "sqlDatabaseProjects.deploy",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "1_dbProjectsFirst"
+        },
+        {
+          "command": "sqlDatabaseProjects.import",
```
Comment (Benjin) [inline extensions/sql-database-projects/package.json:171]: adding schema compare command

File: extensions/sql-database-projects/package.json [hunk 3]
```diff
@@ -119,36 +139,78 @@
...
+          "command": "sqlDatabaseProjects.newScript",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "3_dbProjects_newItem@9"
         },
         {
-          "command": "sqlDatabaseProjects.newFolder",
-          "when": "view == sqlDatabaseProjectsView"
+          "command": "sqlDatabaseProjects.close",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "9_dbProjectsLast"
```
Comment (kisantia) [inline extensions/sql-database-projects/package.json:208]: why does this start with 9?

Comment (Benjin) [reply extensions/sql-database-projects/package.json:208]: the number is the order of the group in the context menu; I wanted to indicate that anyone adding new commands that this should stay last, and new groups should be added in before it.

**Pull request merged**